### DISCUSSION TITLE
"Connect to Codespace: j.sendTelemetryEvent is not a function" whenever I tried to Log-In

### DISCUSSION THREAD

**Discussion by leotrinity16** [community/community#58283] [Codespaces]
### Select Topic Area

Bug

### Body

Hello! I tried logging in to Codespace using VSCode and this error popped up. I can still access on the browser I am now unable to do it with VSCode.

**Comment by the-officialjosh** [comment]
Where is the error it did not send with your question

**Comment by debuglevel** [comment]
This might be a problem with an extension. I had to disable "SARIF Viewer".
